tests/data_context/test_data_context_utils.py

```diff
@@ -49,6 +49,9 @@ def test_load_class_raises_error_when_module_name_is_not_string():
             load_class(bad_input, "great_expectations.datasource")
 
 
+@pytest.mark.filterwarnings(
+    "ignore:SQLAlchemy is not installed*:UserWarning:great_expectations.data_context.util"
+)
```

Comment:
This warning is user-facing but is not relevant to the test at hand. We want to suppress it here but still have it show up in prod.

Reply:
This feels like a good comment to add to the test so we remember